src/core/document.js

```diff
@@ -1815,6 +1819,15 @@ class PDFDocument {
       }
     }
 
+    if (
+      parentRef &&
+      field.get("Subtype") === Name.get("Widget") &&
+      !field.getRaw("Parent")
```

Comment:
Do we actually need to lookup the Parent, or would a simple exists-check suffice?
```suggestion
      !field.has("Parent")
```